gtsam/base/Lie.h

```diff
@@ -17,6 +17,7 @@
  * @author Frank Dellaert
  * @author Mike Bosse
  * @author Duy Nguyen Ta
+ * @author shteren1
```

Comment:
Yotam Stern is my name :)

Reply:
no worries! I already fixed that locally - it will be part of the next commit!